auditbeat/module/auditd/audit_linux_test.go

```diff
@@ -76,24 +78,54 @@ func TestData(t *testing.T) {
 		// Send expected ACKs for initialization
 		returnACK().returnACK().returnACK().returnACK().returnACK().
 		// Send a single audit message from the kernel.
-		returnMessage(userLoginMsg)
+		returnMessage(userLoginMsg).
+		returnMessage(execveMsgs...).
+		returnMessage(acceptMsgs...)
 
 	// Replace the default AuditClient with a mock.
 	ms := mbtest.NewPushMetricSetV2(t, getConfig())
 	auditMetricSet := ms.(*MetricSet)
 	auditMetricSet.client.Close()
 	auditMetricSet.client = &libaudit.AuditClient{Netlink: mock}
 
-	events := mbtest.RunPushMetricSetV2(10*time.Second, 1, ms)
-	if len(events) == 0 {
-		t.Fatal("received no events")
+	events := mbtest.RunPushMetricSetV2(10*time.Second, 3, ms)
+	if len(events) != 3 {
+		t.Fatalf("expected 3 events, but received %d", len(events))
 	}
 	assertNoErrors(t, events)
 
+	assertFieldsAreDocumented(t, events)
+
 	beatEvent := mbtest.StandardizeEvent(ms, events[0], core.AddDatasetToEvent)
 	mbtest.WriteEventToDataJSON(t, beatEvent, "")
 }
 
+// assertFieldsAreDocumented mimics assert_fields_are_documented in Python system tests.
+func assertFieldsAreDocumented(t *testing.T, events []mb.Event) {
+	fieldsYml, err := common.LoadFieldsYaml("../../fields.yml")
```

Comment:
Can you add `mg.Deps(Fields)` to the Go testing parts of the magefile. https://github.com/elastic/beats/blob/9bb6a94eed438971493256e377234dc2d9e9251c/auditbeat/magefile.go#L174-L188

Reply:
done